0-ad-history-mod-package/simulation/templates/units/han/siege_mangonel_packed.xml

```diff
@@ -5,6 +5,11 @@
     <GenericName>Mangonel</GenericName>
     <SpecificName>Tóushí Chē</SpecificName>
     <Icon>units/han/siege_mangonel_packed.png</Icon>
+    <History>
+        Unlike ancient Western civilizations, the ancient Chinese did not use tension or torsion mechanisms for catapults. Instead, their catapults were man-powered: A long horizontal beam was attached to a hinge that was mounted on a framework. A sling was fastened to the end of the beam's back arm. Ropes hung down from the shorter front arm or from a short cross-beam. The team of men operating the catapult stood below the shorter arm. On command, they all pulled the ropes, causing the beam to rotate, the sling to open, and whatever projectile was placed in the sling (usually a stone) to be hurled forward. Such catapults are called mangonels, or traction trebuchets.
+        In contrast, traditional European catapults were powered by the tension of a bow or the torsion of sinew bundles. In the early Middle Ages, however, mangonels also made their way to Europe proved to be superior. This was for several reasons: Firstly, they were simpler to construct and operate. Secondly, because of their mechanical simplicity they could be reloaded faster. For the same reason, they were also way more reliable than the traditional European artillery. For example, in contrast to those, a mangonel's performance was not affected by damp weather conditions. Lastly, it could easily be scaled up or down for different needs by increasing or decreasing the number of men pulling the ropes.
```

Comment:
```suggestion
        In contrast, traditional European catapults were powered by the tension of a bow or the torsion of sinew bundles. In the early Middle Ages, however, mangonels also made their way to Europe, and proved to be far superior. This was for several reasons: Firstly, they were simpler to construct and operate. Secondly, because of their mechanical simplicity, they could be reloaded faster. For the same reason, they were also way more reliable than the traditional European artillery. For example, in contrast to those, a mangonel's performance was not affected by damp weather conditions. Lastly, it could easily be scaled up or down for different needs by increasing or decreasing the number of men pulling the ropes.
```